.gitignore

```diff
@@ -2,3 +2,5 @@ dist/
 node_modules/
 web-ext-artifacts/
 yarn-error.log
+.webpack-cache/
```

Comment:
`.webpack-cache` doesn't exist on this branch. I don't think it needs to be in `.gitignore.

Reply:
i put this there because switching between branches becomes annoying